src/commands/ext-install.ts

```diff
@@ -87,6 +90,11 @@ export default new Command("ext:install [extensionName]")
     // Otherwise, treat the input as an extension reference and proceed with reference-based installation.
     if (isLocalOrURLPath(extensionName)) {
       track("Extension Install", "Install by Source", options.interactive ? 1 : 0);
+      if (options.local) {
```

Comment:
Eeek - not your code, but it looks like we need to add an await at line 92

Reply:
I think this is actually a fire and forget pattern - latency on analytics shouldn't be added to end user.  TIL you can silence this by adding `void`, I can do this for this entire repo

Reply:
https://github.com/firebase/firebase-tools/pull/4215